src/components/ui/Navigation.vue

```diff
@@ -52,5 +56,30 @@ export default class Navigation extends Vue {
     const web3Module = getModule(Web3Module, this.$store)
     enableEthereum(flashesModule, metaMaskModule, web3Module)
   }
+
+  get publicKey() {
+    return this.isAddressDefined ? ethereum.selectedAddress : ''
```

Comment:
I don't think `isAddressDefined()` needs to exist:

    get publicKey() {
        return ethereum.selectedAddress || ''
    }

how does ^ not work for how you are using it here?